webpack/webpack.config.prod.js

```diff
@@ -31,22 +32,31 @@ const deployPlugins = [];
 if (process.env.DEPLOY) {
   deployPlugins.push(new webpack.optimize.UglifyJsPlugin({compressor: {warnings: false}, comments: /(?:)/}));
   deployPlugins.push(new webpack.LoaderOptionsPlugin({comments: false}));
+  deployPlugins.push(new S3Plugin({
```

Comment:
This plugin is pretty great. It'll upload all of our static assets to S3 for us as part of our webpack process.